### PR TITLE
Kirstie's weekly update 15-21 February 2021

### PR DESCRIPTION
**Pull request by KirstieJane**
All reviews welcomed (they make me feel happy!)

### CODE REVIEW

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 1]
```diff
@@ -17,6 +18,102 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+* ASG Theme Leads meeting.
+
+### What did you struggle with?
+
+* Good grief my hay fever 🌳🌳🌳 🤧🥺🤧🥺🤧🥺
+  * I'll find a way to get the right levels of antihistamines in my body for the next 10 months, but....wow this was a frustrating weekend of itchy eyes and not being able to sleep properly ☹️
+* Accidentally stood on a mini landmine.
+  Was a good opportunity to calibrate my bluntness.
+  That was definitely a "woah, I didn't realise I was going into a difficult territory" moment.
+  * Lesson for next time: I think there are probably sensitivities than I realise.
```
Comment (pherterich) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:102]: I think this is missing a "more"?

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 2]
```diff
@@ -17,6 +18,102 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+  * I'm glad I could be of some help.
+  * Also, its motherlode dude.
+    Not motherload.
+
+### What did you achieve
+
+* Grant panel interview!!
+  * It went ok... there are questions we can't answer well but we knew that going in.
+  * There wasn't anything that really tripped us up.
+  * 🤞😬🤞
```
Comment (Arielle-Bennett) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:63]: Fingers crossed!

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 3]
```diff
@@ -17,6 +18,102 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
 
+## Date: 21 February 2021
+
+### What brought you joy this week?
+
+* My greatest joy this week was spending 3 hours working with AL on the community manager job descriptions.
+  * Huge huge shout out to Brene Brown's values list and the exercise of picking just 2 or 3.
+    It's such a hard task but SO valuable!
+    * https://daretolead.brenebrown.com/workbook-art-pics-glossary/
+    * For community managers I picked **stewardship**, **compassion** and **collaboration**.
```
Comment (Arielle-Bennett) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:29]: Very much agree with these three

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md (hunk 1 above)
Comment (KirstieJane) [reply weekly-updates/Kirstie-Whitaker/Weekly-meetings.md]: Thanks @pherterich - fixed!

File: weekly-updates/Kirstie-Whitaker/Weekly-meetings.md [hunk 4]
```diff
@@ -17,6 +18,102 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
...
+* Really glad SM was able to come to the Turing Way collaboration cafe!! ☕
+* LOVE the help that IS was able to provide for ME! :heart_eyes:
+  * https://github.com/WhitakerLab/WhitakerLabProjectManagement/blob/master/weekly-updates/Maria-Eriksson/Mentoring-IS.md#date-11-february-2021
+* Knowledge share this week made me a bit teary to actually see the work progressing, and hear how important it still is for clinicians.
+* AB sent the TPS newsletter in its new format!
+  * So beautiful!
+  * http://elink.turing.ac.uk/m/1/59207972/p1-b21050-e368bf4a814a4324a28ac2cb84e12511/1/13/843f7bf5-d1bc-4395-b06f-13236425037e
+* Maybe not exactly joy, but I helped a few friends through difficult frustrations with unsupportive colleagues (not at Turing) this week.
+  * I'm glad I could be of some help.
+  * Also, its motherlode dude.
```
Comment (Islast) [inline weekly-updates/Kirstie-Whitaker/Weekly-meetings.md:55]: I never knew that! ✨

Comment (KirstieJane) [reply weekly-updates/Kirstie-Whitaker/Weekly-meetings.md]: It's actually supposed to be 2 words.... fixed now. And a link to https://en.wikipedia.org/wiki/Mother_lode

**Review by Islast — APPROVED** [review]
amazing stuff Kirstie ✨

**Pull request merged**